doc/api/fs.md

```diff
@@ -494,10 +494,6 @@ changes:
 
```

Comment:
You should modify the YAML to add this change (see the lines right above this comment)

Reply:
Right, but what do I put for `version`?

Reply:
Nevermind, it's `REPLACEME`